app/views/IndexView.scala.html

```diff
@@ -20,120 +20,142 @@
 @import utils.CurrencyFormatter.currencyFormat
 @import java.time.format.DateTimeFormatter
+@import java.time.LocalDate
+@import models.exclusions.ExcludedTrader
 
 @this(
-        layout: templates.Layout,
-        govukButton: GovukButton
+layout: templates.Layout,
+govukButton: GovukButton
 )
 @(
-        businessName: String,
-        vrn: String,
-        returnsViewModel: ReturnsViewModel,
-        paymentsViewModel: PaymentsViewModel,
-        paymentError: Boolean
+businessName: String,
+vrn: String,
+returnsViewModel: ReturnsViewModel,
+paymentsViewModel: PaymentsViewModel,
+paymentError: Boolean,
+excludedTrader: Option[ExcludedTrader]
 )(implicit request: Request[_], messages: Messages)
 
 @layout(
-    pageTitle = titleNoForm(messages("index.title")),
-    showBackLink = false,
-    fullWidth = true
+pageTitle = titleNoForm(messages("index.title")),
+showBackLink = false,
+fullWidth = true
 ) {
 
-    @if(paymentError) {
-        <div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
-            <div class="govuk-notification-banner__header">
-                <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
-                @messages("notificationBanner.title")
-                </h2>
-            </div>
-            <div class="govuk-notification-banner__content">
-                <p class="govuk-notification-banner__heading">
-                @messages("notificationBanner.text")
-                </p>
-            </div>
-        </div>
-    }
+@if(paymentError) {
+<div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title"
+     data-module="govuk-notification-banner">
+    <div class="govuk-notification-banner__header">
+        <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+            @messages("notificationBanner.title")
+        </h2>
+    </div>
+    <div class="govuk-notification-banner__content">
+        <p class="govuk-notification-banner__heading">
+            @messages("notificationBanner.text")
+        </p>
+    </div>
+</div>
+}
 
-    <h1 class="govuk-heading-l">
-        @messages("index.heading")
-        <span class="govuk-caption-m">@businessName</span>
-        <span class="govuk-caption-m">@messages("index.vrn", vrn)</span>
-    </h1>
+<h1 class="govuk-heading-l">
+    @messages("index.heading")
+    <span class="govuk-caption-m">@businessName</span>
+    <span class="govuk-caption-m">@messages("index.vrn", vrn)</span>
+</h1>
 
-    <p class="govuk-body">@messages("index.paragraph")</p>
+<div class="govuk-warning-text">
+    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+    <strong class="govuk-warning-text__text">
+        <span class="govuk-warning-text__assistive">Warning</span>
+        @if(excludedTrader.nonEmpty) {
+            @if(excludedTrader.get.exclusionReason == 4) {
```

Comment:
You can simplify this by using a case/match statement

Reply:
If statement is simplified with case/match statement